auth_matrix/admin_views.py

```diff
@@ -6,6 +6,7 @@
 
 User = get_user_model()
 
+
 @method_decorator(staff_member_required, name="dispatch")
```

Comment:
we tend to follow a stricter approach to add proper permission checks - staff user does not automatically mean you should get access to this information! The ticket also mentions that it should be connected to the permission to view users.

You can look at https://github.com/open-formulieren/open-forms/blob/master/src/openforms/config/admin_views.py#L11 for inspiration and you typically also want to wrap this properly with `admin.site.admin_view`: https://github.com/open-formulieren/open-forms/blob/b0feb8aa60d10c16ede0873420570a5d4668d3e2/src/openforms/admin/urls.py#L31